modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -1385,16 +1358,43 @@ private void _assertNotificationQueueEntryTermValues(
 		NotificationQueueEntry notificationQueueEntry =
 			notificationQueueEntries.get(0);
 
-		assertTermValues(
-			TransformUtil.transform(
-				expectedTermValues, this::parseTermValueToString),
-			Arrays.asList(
-				StringUtil.split(notificationQueueEntry.getBody(), delimiter)));
+		Assert.assertEquals(expectedBody, notificationQueueEntry.getBody());
 
 		notificationQueueEntryLocalService.deleteNotificationQueueEntry(
 			notificationQueueEntry);
 	}
 
+	private String _formatDate(Date date) {
+		if (date == null) {
+			return StringPool.BLANK;
+		}
+
+		DateFormat dateFormat = DateFormatFactoryUtil.getSimpleDateFormat(
+			DateTimeFormatterBuilder.getLocalizedDateTimePattern(
+				FormatStyle.SHORT, FormatStyle.SHORT, IsoChronology.INSTANCE,
+				LocaleUtil.US),
+			LocaleUtil.US);
+
+		return dateFormat.format(date);
+	}
+
+	private String _getCommerceOrderExpectedBody(CommerceOrder commerceOrder)
```

Comment:
I would rename it to something a little more explicit like  

1. `_getCommerceOrderNotificationQueueEntryBody` and 
2. `_getObjectEntryNotificationQueueEntryBody` 

or inline

```
private void _assertNotificationQueueEntryBody(Object object)
	throws Exception {

	List<NotificationQueueEntry> notificationQueueEntries =
		notificationQueueEntryLocalService.getNotificationEntries(
			NotificationConstants.TYPE_EMAIL,
			NotificationQueueEntryConstants.STATUS_SENT);

	Assert.assertEquals(
		notificationQueueEntries.toString(), 1,
		notificationQueueEntries.size());

	String expectedBody = null;

	if (object instanceof CommerceOrder) {
		CommerceOrder commerceOrder = (CommerceOrder) object;

		AccountEntry accountEntry = commerceOrder.getAccountEntry();

		expectedBody =
			StringUtil.merge(
				Arrays.asList(
					String.valueOf(accountEntry.getAccountEntryId()),
					String.valueOf(commerceOrder.isInactive()),
					commerceOrder.getName(),
					_formatDate(commerceOrder.getOrderDate()),
					String.valueOf(commerceOrder.isPending()),
					_formatDate(commerceOrder.getRequestedDeliveryDate()),
					_formatDate(commerceOrder.getStatusDate())),
				StringPool.NEW_LINE);
	}
	else if (object instanceof ObjectEntry) {
		ObjectEntry objectEntry = (ObjectEntry) object;

		com.liferay.object.model.ObjectEntry serviceBuilderObjectEntry =
			_objectEntryLocalService.getObjectEntry(objectEntry.getId());

		expectedBody = StringUtil.merge(
			Arrays.asList(
				LanguageUtil.getLanguageId(LocaleUtil.US),
				serviceBuilderObjectEntry.getUserName(),
				_formatDate(serviceBuilderObjectEntry.getCreateDate()),
				serviceBuilderObjectEntry.getExternalReferenceCode(),
				_formatDate(serviceBuilderObjectEntry.getModifiedDate()),
				String.valueOf(serviceBuilderObjectEntry.getObjectEntryId()),
				_formatDate(serviceBuilderObjectEntry.getLastPublishDate()),
				WorkflowConstants.getStatusLabel(
					serviceBuilderObjectEntry.getStatus()),
				"true", "9/25/24 12:00 AM", "2024-09-25T00:00",
				"test@liferay.com", "12345", "123456789",
				"listTypeEntry1Value,listTypeEntry2Value",
				"listTypeEntry1Value", "", "textObjectFieldValue",
				_portal.getPortalURL(
					ServiceContextThreadLocal.getServiceContext(
					).getRequest())),
			StringPool.NEW_LINE);
	}
	else {
		expectedBody = (String) object;
	}

	NotificationQueueEntry notificationQueueEntry =
		notificationQueueEntries.get(0);

	Assert.assertEquals(expectedBody, notificationQueueEntry.getBody());

	notificationQueueEntryLocalService.deleteNotificationQueueEntry(
		notificationQueueEntry);
}
```